launch.go

```diff
@@ -147,7 +147,7 @@ func quickLauncherHandler(w http.ResponseWriter, r *http.Request) {
 
 	urlValues.Add("ru_ref", authentication.GetDefaultValues()["ru_ref"])
 	urlValues.Add("collection_exercise_sid", uuid.NewV4().String())
-	urlValues.Add("case_id", uuid.NewV4().String())
```

Comment:
Do we want to keep this anyway since it is optional now in runner.

Reply:
The card seems to specify that it should replace questionnaire_id

Reply:
The card says it's no longer mandatory. therefore we should keep it and only send it if a value is provided.